packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -107,6 +110,91 @@ export class FromCloudFormation {
     return ret;
   }
 
+  public static parseCreationPolicy(policy: any): CfnCreationPolicy | undefined {
+    if (policy == null) { return undefined; }
```

Comment:
Shouldn't we return `undefined` for any type that's not an 'object'?

Reply:
Sure.